Fabulous.XamarinForms/src/Fabulous.XamarinForms/Xamarin.Forms.Core.json

```diff
@@ -2822,6 +2791,137 @@
       "primaryConstructorMembers": [
         "Items"
       ]
+    },
+    {
+      "type": "Xamarin.Forms.IndicatorView",
+      "name": "Indicator",
+      "properties": [
+        {
+          "source": "Count"
+        },
+        {
+          "source": "HideSingle"
+        },
+        {
+          "source": "IndicatorLayout"
+        },
+        {
+          "source": "IndicatorSize"
+        },
+        {
+          "source": "IndicatorTemplate"
+        },
+        {
+          "source": "IndicatorShape"
+        },
+        {
+          "source": "ItemsSource"
+        },
+        {
+          "source": "MaximumVisible"
+        },
+        {
+          "source": "Position"
+        },
+        {
+          "source": "SelectedIndicatorColor"
+        }
+      ]
+    },
+    {
+      "type": "Xamarin.Forms.SwipeItem",
+      "name": "SwipeItem",
+      "properties": [
+        {
+          "source": "BackgroundColor"
+        }
+      ],
+      "events": [
+        {
+          "source": "Invoked"
+        }
+      ]
+    },
+    {
+      "type": "Xamarin.Forms.SwipeItemView",
+      "name": "SwipeItemView",
+      "properties": [
+        {
+          "source": "Command"
+        }
+      ],
+      "events": [
+        {
+          "source": "Invoked"
+        }
+      ]
+    },
+    {
+      "type": "Xamarin.Forms.SwipeItems",
```

Comment:
This one can be a problem for incremental updates. It's a list in itself.
We will need to make sure it behaves correctly.